### DISCUSSION TITLE
add global position for all objects

### DISCUSSION THREAD

**Discussion by Q210108** [opentoonz/opentoonz#6454] [Ideas]
At the moment, there are temporary XY position : when you move an object at x:5 y:8 for instance, and when you clic on the object again, the position is lost. What i suggest is a new global position that is not temporary
![image](https://user-images.githubusercontent.com/100626161/158463863-68000abf-05f6-4790-823f-fc4b0a50800c.png)

**Comment by RodneyBaker** [comment]
You may want to explore the Schematic where globally positioned 'objects' can be set up that remain static and then can be moved via various interpolations to achieve a number of different movements... even lock movement down.  Pegs are an example of this but many users favor using parent/child setups using null (empty) columns as they have some capabilities Pegs don't have.  The Table is perhaps the ultimate global positioning reference.   

In what way does setting up objects via Schematic not address your issues with global positioning?
